index.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Coolors Fest</title>
+    <link rel="stylesheet" href="./assets/Styles/main.css">
+  </head>
+  <body>
+    <div id="menuMobile">
+      <div id="menuDisplay" class="hidden">
+        <i id="closeBtn" >
+          X
+        </i>
+        <a href="./about.html">About</a>
+      </div>
+      <a 
+        id="hambMenu"
+        href="#">
+        <img 
+          src="./assets/svg/hamb-menu.svg" 
+          alt="Menu icon">
+      </a>      
+    </div>
+    <div id="socialBar">
+      <div class="social-icons">
+        <img src="./assets/svg/icon-facebook.svg" alt="facebook icon">
+        <img src="./assets/svg/icon-twitter.svg" alt="twitter icon">
+        <a href="#">English</a>
+        <a href="#">My page</a>
+        <a href="#">Log out</a>
+      </div>
+    </div>
+    <nav id="deskNav">
```

Comment:
- [ ] **The navbar on the homepage doesn't match the original design. Please fix it.**

Original Design|Cloned Design
:-------------------:|:------------------:
![image](https://user-images.githubusercontent.com/59168430/125124948-86adcf80-e116-11eb-8c91-25b3164199ab.png)|![image](https://user-images.githubusercontent.com/59168430/125124786-464e5180-e116-11eb-8faf-c6f89e15aba0.png)

In the original design, the logo is aligned on the left and the nav is aligned on the right. On the cloned page, the nav is spanned over the entire element. Please fix it. Take the nav on your about page, it is perfect please use the same nav on both pages.

Reply:
Fixed :heavy_check_mark: